backend/tests/unit/services/oauth/test_oauth_service.py

```
@@ -72,54 +71,59 @@ async def test_get_authorization_url_success(self, git_provider_service, redirec
 
     @pytest.mark.asyncio
     async def test_exchange_code_for_token_success(self, git_provider_service, auth_code, redirect_uri, test_state, github_token_response):
         """Test successful token exchange."""
-        # Store state first
-        git_provider_service.state_manager.store_state(
-            state=test_state,
-            provider="github",
-            redirect_uri=redirect_uri
+        # Mock the state manager to return valid state
+        from database.models import OAuthState as DBOAuthState
+        from datetime import datetime, UTC, timedelta
```

Ruff (0.14.1)

Redefinition of unused `datetime` from line 7

Remove definition: `datetime`

(F811)